src/Service/AbstractService.php

```diff
@@ -92,7 +92,7 @@ public function getEntitiesByField($class, $fields, $currentPage, $resultsPerPag
     /**
      * @return SessionInterface
      */
```

Comment:
Services must not expose class attributes through getters. IMO you broke encapsulation.